packages/react/src/components/DatePicker/DatePicker.styles.ts

```diff
@@ -75,5 +79,39 @@ export const styles = (props: IDatePickerStyleProps): IDatePickerStyles => {
         marginTop: 5,
       },
     ],
+    readOnlyTextfield: [
```

Comment:
Can these styles be exported from TextField.styles.ts and mostly reused here, or are they too different?

Reply:
I think they're different in the sense that having styling needed by `DatePicker` stored in `TextField`'s stylesheet doesn't seem quite right 🤔. Also from a usage standpoint, this `readOnlyTextfield` style is used in a span which is nested in a `div` within `DatePicker`'s `onRenderReadOnlyInput` function; so if I were to export those styles from `TextField.styles.ts`, I'm not quite sure how to incorporate that with this pattern 😥